_posts/2016-09-29-Rust-1.12.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: post
+title: "Announcing Rust 1.12.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.12.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.12.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.12.0][notes] on GitHub.
+1361 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1120-2016-09-29
+
+### What's in 1.12.0 stable
+
+Oh my god! So much stuff!
+
+With 1.12 stable, one of the first visible changes you will notice is the new
+error message format emitted by `rustc`. We've [previously talked] about this
+format and this is the first release where they are broadly available. These
+error messages are a result of the effort of many hours of [volunteer effort] to
+design, test, and update every one of `rustc`s errors to the new format. We're
+excited to see what you think of them:
+
+![A new mismatched trait
+error](/images/2016-09-Rust-1.12/mismatched-trait-error.png)
```

Comment:
Do we want to use the same trait error in both this section and the expanded section below?  Might read better to have pictures of a different kind of error here.